servicex/values.yaml

```diff
@@ -10,6 +10,7 @@ secrets:
 # want to have the option of delivering results as parquet files
 objectStore:
   enabled: true
+  internal: false
   publicURL:
```

Comment:
This should default to `true` to keep existing behavior. It will break all current deployments otherwise

Reply:
Pushed a fix